app/src/main/java/endava/codebase/android/movieapp/ui/component/FavoriteButton.kt

```diff
@@ -0,0 +1,45 @@
+package endava.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.material.Checkbox
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import endava.codebase.android.movieapp.R
+
+@Composable
+fun FavoriteButton(
+    isFavorite: Boolean,
+    onCheckedChange: (Boolean) -> Unit
+) {
+    Box(contentAlignment = Alignment.TopStart) {
+        Box(contentAlignment = Alignment.Center) {
```

Comment:
You can remove both boxes. First `Box` should be removed because it is just a wrapper for the second `Box` and is actually just setting alignment on the screen, which is something that should be done when calling `FavoriteButton` component via modifier. Second `Box` is not needed because when you remove the `Checkbox`, there will only be one component left in that box.

Reply:
I need to have one box because it does not let me use .align(Alignment.TopStart) if the control is not inside box.

Reply:
This should not be a problem because alignment should be defined when component is called. In this case, you would remove `Box` and `.align(Alignment.TopStart)` from modifier, and set box in `MovieCard` and pass alignment via modifier.